docs/pages/collectorintro.md

```diff
@@ -0,0 +1,35 @@
+
+#Introduction
+
+Refocus makes it easy for users to understand the real-time status of their systems. To date, achieving this has required users to develop API integrations in order to push Samples into Refocus. This process had a few limitations: it required developers to write API integrations, and, at times, meant teams had to operate an extra 'always-on' service. 
+
+Collectors are designed to tackle these issues and make it easier than ever to leverage all the capabilities of Refocus!
+
+#So, what is a Collector 
+
+A Collector is an out-of-the-box application (running outside of Refocus) which collects data from different data sources and sends Samples to Refocus. It does this by leveraging user-provided config files called a “Sample Generators”.
+Most users will never have to maintain a collector (unless you choose to deploy your own), they'll just write Sample Generators.
```

Comment:
I don't think we can say "Most users will never have to maintain a collector". The main point to convey is that a single Collector can handle multiple Sample Generators.

Reply:
I think this is an important point to make. Most people reading this won't need to interact directly with collectors at all - they need to know that. That's the main point of the system!